### ISSUE TITLE
NC28 Action button with three dots missing in all browsers

### ISSUE THREAD

**Issue by dwprod38** [nextcloud/server#42414]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

After updating from NC27.1.5 to   28.0.0.11. Action button with three dots is missing in all browsers after selected multiple files… impossible to move or copy them in folder…

### Steps to reproduce

1. Open browser of your choice
2. In nextcloud page, open a folder and select more than one file
3. Action button with three dots not appeared… 

### Expected behavior

Action button with three dots is displayed  in all browsers after selected multiple files to move or copy them in other folder…

worked fine with NC27.1.5…

### Installation method

Community NextcloudPi appliance

### Nextcloud Server version

28

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.2

### Web server

Apache (supported)

### Database engine version

MySQL

### Is this bug present after an update or on a fresh install?

Upgraded to a MAJOR version (ex. 22 to 23)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
After entering command "sudo -u www-data php occ config:list system", got this message:

{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "default_phone_region": "FR",
        "trusted_domains": {
            "0": "localhost",
            "7": "nextcloudpi",
            "5": "nextcloudpi.local",
            "8": "nextcloudpi.lan",
            "3": "nextcloudpi",
            "11": "***REMOVED SENSITIVE VALUE***",
            "1": "***REMOVED SENSITIVE VALUE***",
            "2": "***REMOVED SENSITIVE VALUE***",
            "14": "nextcloudpi",
            "20": "***REMOVED SENSITIVE VALUE***
        },
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "28.0.0.11",
        "overwrite.cli.url": "https:\/\/nextcloudpi.local\/",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0,
            "password": "***REMOVED SENSITIVE VALUE***"
        },
        "tempdirectory": "\/media\/myCloudDrive\/ncdatadwprod38\/data\/tmp",
        "mail_smtpmode": "sendmail",
        "mail_smtpauthtype": "LOGIN",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "preview_max_x": "2048",
        "preview_max_y": "2048",
        "jpeg_quality": "60",
        "overwriteprotocol": "https",
        "maintenance": false,
        "logfile": "\/media\/myCloudDrive\/ncdatadwprod38\/data\/nextcloud.log",
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "data-fingerprint": "d22797f990fc929ee825dd7233807d69",
        "loglevel": "2",
        "log_type": "file",
        "theme": "",
        "app_install_overwrite": [
            "camerarawpreviews",
            "workflow_media_converter"
        ]
    }
}
```

### List of activated Apps

```shell
After entering command "sudo -u www-data php occ app:list", got this message:

Enabled:
  - activity: 2.20.0
  - calendar: 4.6.0
  - circles: 28.0.0-dev
  - cloud_federation_api: 1.11.0
  - comments: 1.18.0
  - contacts: 5.5.0
  - contactsinteraction: 1.9.0
  - dashboard: 7.8.0
  - dav: 1.29.1
  - federatedfilesharing: 1.18.0
  - federation: 1.18.0
  - files: 2.0.0
  - files_pdfviewer: 2.9.0
  - files_reminders: 1.1.0
  - files_sharing: 1.20.0
  - files_trashbin: 1.18.0
  - files_versions: 1.21.0
  - firstrunwizard: 2.17.0
  - logreader: 2.13.0
  - lookup_server_connector: 1.16.0
  - nextcloud_announcements: 1.17.0
  - notes: 4.9.1
  - notifications: 2.16.0
  - notify_push: 0.6.6
  - oauth2: 1.16.3
  - password_policy: 1.18.0
  - photos: 2.4.0
  - privacy: 1.12.0
  - provisioning_api: 1.18.0
  - recommendations: 2.0.0
  - related_resources: 1.3.0
  - serverinfo: 1.18.0
  - settings: 1.10.0
  - sharebymail: 1.18.0
  - support: 1.11.0
  - survey_client: 1.16.0
  - systemtags: 1.18.0
  - tasks: 0.15.0
  - text: 3.9.1
  - theming: 2.3.0
  - twofactor_backupcodes: 1.17.0
  - updatenotification: 1.18.0
  - user_status: 1.8.1
  - viewer: 2.2.0
  - weather_status: 1.8.0
  - workflow_media_converter: 1.8.4
  - workflowengine: 2.10.0
Disabled:
  - admin_audit: 1.18.0
  - bruteforcesettings: 2.8.0
  - camerarawpreviews: 0.8.4 (installed 0.8.4)
  - encryption: 2.16.0
  - files_external: 1.20.0
  - news: 24.0.0 (installed 24.0.0)
  - nextcloudpi: 0.0.2 (installed 0.0.1)
  - previewgenerator: 99.99.99 (installed 99.99.99)
  - suspicious_login: 6.0.0
  - twofactor_totp: 10.0.0-beta.2
  - user_ldap: 1.19.0
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

20 déc. 2023, 21:39:48	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

20 déc. 2023, 21:39:48	

Error	webdav	
InvalidUriException
Invalid, or could not parse URI

20 déc. 2023, 16:49:33	

Error	webdav	
InvalidUriException
Invalid, or could not parse URI

20 déc. 2023, 16:48:48	

Error	webdav	
InvalidUriException
Invalid, or could not parse URI

20 déc. 2023, 16:47:43	

Error	webdav	
InvalidUriException
Invalid, or could not parse URI

20 déc. 2023, 16:47:09	

Error	webdav	
InvalidUriException
Invalid, or could not parse URI

20 déc. 2023, 16:43:18	

Error	webdav	
InvalidUriException
Invalid, or could not parse URI

20 déc. 2023, 16:42:55	

Error	webdav	
InvalidUriException
Invalid, or could not parse URI

20 déc. 2023, 16:39:53	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

20 déc. 2023, 11:33:16	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

20 déc. 2023, 10:23:36	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

18 déc. 2023, 15:04:32	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

18 déc. 2023, 15:04:32	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

18 déc. 2023, 15:04:32	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

18 déc. 2023, 08:40:25	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

18 déc. 2023, 08:40:25	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

18 déc. 2023, 08:40:25	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

17 déc. 2023, 17:42:18	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

17 déc. 2023, 17:42:18	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

17 déc. 2023, 12:28:00	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

17 déc. 2023, 12:28:00	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

17 déc. 2023, 12:28:00	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

16 déc. 2023, 23:22:31	

Error	no app in context	
Exception
HMAC does not match.
Could not decrypt or decode encrypted session data

16 déc. 2023, 23:22:31	

Info	updater	
\OC\Updater::resetLogLevel: Reset log level to Warning(2)

14 déc. 2023, 17:59:27	

Info	updater	
\OC\Updater::maintenanceDisabled: Turned off maintenance mode

14 déc. 2023, 17:59:27	

Info	updater	
\OC\Updater::updateEnd: Update successful

14 déc. 2023, 17:59:27	

Info	updater	
\OC\Updater::finishedCheckCodeIntegrity: Finished code integrity check

14 déc. 2023, 17:59:27	

Info	updater	
\OC\Updater::startCheckCodeIntegrity: Starting code integrity check...

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Queue a job to generate metadata

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Add AI tasks cleanup job

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Add possibly missing system config

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Clean up abandoned apps

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Add token cleanup job

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Add background job to set the lookup server share state for users

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Repair DAV shares

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Queue a one-time job to check for user uploaded certificates

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Add job to cleanup the bruteforce entries

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Remove old dashboard app config data

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Check encryption key format

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Keep legacy encryption enabled

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Reset generated avatar flag

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Clear access cache of projects

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: No need to remove link shares.

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Remove potentially over exposing share links

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Add background job to cleanup login flow v2 tokens

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Cleanup invalid photocache files for carddav

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Queue a one-time job to cleanup old backups of the updater

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Add preview background cleanup job

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Clear every generated avatar

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: JS cache cleared

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: Image cache cleared

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Clear frontend caches

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Add log rotate job

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Repair language codes

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: No mounts updated

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Fix potential broken mount points

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: Delete clients (and their related access tokens) with the redirect_uri starting with oc:// or ending with *

14 déc. 2023, 17:59:14	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: Update the oauth2_clients table schema.

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: Update the oauth2_access_tokens table schema.

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Migrate oauth2_clients table to nextcloud schema

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Add preview cleanup background jobs

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: Repair step already executed

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Add move avatar background job

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: .step file moved to .step-previous-update

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: .step-previous-update removed

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: .step file exists

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Move .step file of updater to backup location

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Repair invalid shares

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: 0 tags with no entries have been removed.

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: 0 tag entries for deleted tags have been removed.

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: 0 tags for delete files have been removed.

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: 0 tags of deleted users have been removed.

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Clean tags and favorites

14 déc. 2023, 17:59:13	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Repair mime types

14 déc. 2023, 17:59:12	

Info	updater	
OC\Repair\Events\RepairInfoEvent: Repair info: All tables already have the correct collation -> nothing to do

14 déc. 2023, 17:59:12	

Info	updater	
OC\Repair\Events\RepairStepEvent: Repair step: Repair MySQL collation

14 déc. 2023, 17:59:12
```

**Comment by susnux** [comment]
I am not sure I understand the problem, but the actions for copy and move are placed outside the "..." menu, the "..." is still there and contains e.g. the download action.

![image](https://github.com/nextcloud/server/assets/1855448/fd9ca135-c4b6-41c7-83f9-3376a020c0d0)

**Comment by dwprod38** [comment]
Hello @susnux, 

With NC28.0.0.11, "Move or copy" and "..." buttons didn't exist, only "+New" and " Delete" icons.

Since update to NC28.0.1 --> New Icons "Move or copy" and "..." appeared and working fine. The "drag and drop" file seems not working since NC28.0.0.11 (blinking phenomenon "Drop icon"  zone). After releasing mouse button: the dropped file appeared in browser page instead of downloading. Here a short video example of Drag and Drop issue (MacOS Sierra Chrome version 103.0.5060.134):

https://github.com/nextcloud/server/assets/95407859/ee37aaa5-074b-4ecb-99b5-116613e1c084

**Comment by susnux** [comment]
Hey @dwprod38 thank you for the information, so 28.0.1 solved this issue. Regarding:

> The "drag and drop" file seems not working since NC28.0.0.11 (blinking phenomenon "Drop icon" zone). After releasing mouse button: the dropped file appeared in browser page instead of downloading

This is fixed with https://github.com/nextcloud/server/pull/42409 and will be released with 28.0.2
